python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -248,6 +248,10 @@ def store_asset_event(self, event: EventLogEntry, event_id: int):
         # https://github.com/dagster-io/dagster/issues/3945
 
         values = self._get_asset_entry_values(event, event_id, self.has_asset_key_index_cols())
+
+        if not values:
+            return
+
```

Comment:
possible point of discussion - I am not currently updating the "last_run_id" or "last_materialization_timestamp" columns on this new event type, although i certainly could (both of those columns are currently updated on planned and materialization events)

It looks like the main place we use the timestamp column is in the context of wipes